website/docs/ephemeral-resources/outputs.html.markdown

```diff
@@ -0,0 +1,63 @@
+---
+layout: "tfe"
+page_title: "Terraform Enterprise: Ephemeral: tfe_outputs"
+description: |-
+  Get output values from another organization/workspace without writing
+  sensitive data to state.
+---
+
+# Ephemeral: tfe_organization_token
+
+Terraform ephemeral resource for managing a TFE organization token. This
+resource is used to generate a new organization token that is guaranteed not to
+be written to state. Since organization tokens are singleton resources, using this ephemeral resource will replace any existing organization token, including those managed by `tfe_organization_token`.
```

Comment:
I think this description is copied from another resource...

Reply:
Addressed in 7d7be8ec.